share/spack/gitlab/cloud_pipelines/stacks/radiuss-aws-aarch64/spack.yaml

```diff
@@ -2,15 +2,15 @@ spack:
   view: false
   packages:
     all:
-      providers:
-        blas:
-          - openblas
-        mkl:
-          - intel-oneapi-mkl
-        mpi:
-          - openmpi
-          - mpich
-      variants: +mpi
+      require:
+      - "target=aarch64"
+      - "+mpi"
+    blas:
+      require:
+      - openblas
+    mpi:
+      require:
+      - openmpi
```

Comment:
I noticed in this stack you only add `all: require: ["target=aarch64"]`, while everywhere else it seems you make sure to also include, e.g. `target: ["aarch64"]`.  Is that because this stack includes the `target: ["aarch64"]` in it's `definitions`?

Reply:
Just overlooked. Having:
```yaml
all:
  target: ["x86_64_v3"]
```
makes that target the "best" in terms of optimization weights, so clingo would try to chose it first, even if we don't require it.